languagetool-language-modules/pt/src/main/java/org/languagetool/rules/pt/PortugueseClicheRule.java

```diff
@@ -44,16 +41,19 @@ public class PortugueseClicheRule extends AbstractSimpleReplaceRule2 {
 
   public static final String PORTUGUESE_CLICHE_RULE = "PT_CLICHE_REPLACE";
 
-  private static final String FILE_NAME = "/pt/cliches.txt";
+//  private static final String FILE_NAME = "/pt/cliches.txt";
```

Comment:
Better to delete this line instead of commenting it out (it's still in the git history, should we ever need it again).